<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>QgsUValDialogBase</class>
 <widget class="QDialog" name="QgsUValDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="2" colspan="2" >
    <widget class="Q3ListBox" name="mClassBreakBox" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="Q3WidgetStack" name="mSymbolWidgetStack" >
     <widget class="QWidget" name="WStackPage" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>153</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>210</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QComboBox" name="mClassificationComboBox" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="mClassVarLabel" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
     <property name="text" >
      <string>Classification Field:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3WidgetStack</class>
   <extends></extends>
   <header>q3widgetstack.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends></extends>
   <header>q3listbox.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
